enterprise/releases/2021-12-0.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: '2021.12.0 Release'
+backlink_href: /docs/enterprise/releases
+backlink_title: 'pganalyze Enterprise Edition: Release Changelog'
+---
+
+**Docker tag:** `quay.io/pganalyze/enterprise:v2021.12.0`
+
+## Notes
+
+* This release requires a database migration when coming from older releases, be sure to follow the [recommended upgrade steps](/docs/enterprise/upgrade)
+* This release introduces the [pganalyze Index Advisor](/blog/introducing-pganalyze-index-advisor), note that due to the additional background processing you may need additional worker capacity
+
+## New features
+
+* Index Advisor
+  - This is a major new feature - you can read more about it in our blog posts:
+    - [A better way to index your Postgres database: pganalyze Index Advisor](/blog/introducing-pganalyze-index-advisor)
+    - [How we deconstructed the Postgres planner to find indexing opportunities](/blog/deconstructing-the-postgres-planner)
+* Single Sign-On with SAML, supporting Okta & others
+  - You can now enable Single Sign-On using the Integrations page (under Settings)
+  - Once enabled the main login page will ask users to login with SSO instead
+  - Note that existing users with conflicting email addresses will need to change their email or have their user removed before SSO logins are permitted
+* Add updateServer and deleteServer GraphQL mutations
+  - These can be used for automating updating the pganalyze configuration, e.g. to adjust the database hostname when using the local built-in collector
```

Comment:
It occurs to me we have not documented these, right? But that can wait.

Reply:
Yeah, they are not documented yet - we can adjust the release notes once we do so